tests/business-critical-integration/integration-test-app/IterableSDK-Integration-Tester/Tests/EmbeddedMessageIntegrationTests.swift

```diff
@@ -4,541 +4,163 @@ import UserNotifications
-        // Refresh embedded message view
-        let refreshEmbeddedButton = app.buttons["refresh-embedded-view"]
-        XCTAssertTrue(refreshEmbeddedButton.waitForExistence(timeout: standardTimeout))
-        refreshEmbeddedButton.tap()
-        
-        // Verify updated content
-        let updatedContentText = app.staticTexts["updated-embedded-content"]
-        XCTAssertTrue(updatedContentText.waitForExistence(timeout: standardTimeout))
-        
-        screenshotCapture.captureScreenshot(named: "updated-embedded-content")
-        
-        // Verify content update metrics
-        validateMetrics(eventType: "embeddedMessageUpdate", expectedCount: 1)
-    }
-    
-    func testEmbeddedMessageNetworkHandling() {
-        // Test embedded message behavior with network connectivity issues
-        
-        validateSDKInitialization()
-        
-        // Enable embedded messaging
-        let enableEmbeddedButton = app.buttons["enable-embedded-messaging"]
-        XCTAssertTrue(enableEmbeddedButton.waitForExistence(timeout: standardTimeout))
-        enableEmbeddedButton.tap()
-        
-        // Make user eligible
-        let makeEligibleButton = app.buttons["make-user-eligible"]
-        makeEligibleButton.tap()
-        
-        // Enable network failure simulation
-        let enableNetworkFailureButton = app.buttons["enable-network-failure-simulation"]
-        XCTAssertTrue(enableNetworkFailureButton.waitForExistence(timeout: standardTimeout))
-        enableNetworkFailureButton.tap()
-        
-        // Attempt to create embedded campaign while network is down
-        let createEmbeddedOfflineButton = app.buttons["create-embedded-offline"]
-        XCTAssertTrue(createEmbeddedOfflineButton.waitForExistence(timeout: standardTimeout))
-        createEmbeddedOfflineButton.tap()
-        
-        // Verify offline handling
-        let offlineHandledIndicator = app.staticTexts["embedded-offline-handled"]
-        XCTAssertTrue(offlineHandledIndicator.waitForExistence(timeout: standardTimeout))
-        
-        screenshotCapture.captureScreenshot(named: "embedded-offline-handled")
-        
-        // Restore network
-        let restoreNetworkButton = app.buttons["restore-network"]
-        XCTAssertTrue(restoreNetworkButton.waitForExistence(timeout: standardTimeout))
-        restoreNetworkButton.tap()
-        
-        // Retry embedded message creation
-        let retryEmbeddedCreationButton = app.buttons["retry-embedded-creation"]
-        XCTAssertTrue(retryEmbeddedCreationButton.waitForExistence(timeout: standardTimeout))
-        retryEmbeddedCreationButton.tap()
-        
-        // Verify successful retry
-        let retrySuccessIndicator = app.staticTexts["embedded-creation-retry-success"]
-        XCTAssertTrue(retrySuccessIndicator.waitForExistence(timeout: networkTimeout))
-        
-        screenshotCapture.captureScreenshot(named: "embedded-network-retry-success")
+    func testEmbeddedMessage() {
```

Comment:
Optional - I realized we should have a comment section right above this primary method which tells whats expected for it to run. 
Like a proper project with placement already created and a campaign which works for a dynamic list which looks for isPremium flag under user fields.